teachertool/src/components/styling/CriteriaInstanceDisplay.module.scss

```diff
@@ -49,6 +49,19 @@
                 border: solid 1px var(--pxt-page-foreground);
                 padding: 0.4rem;
 
+                .block-readable-name {
+                    .block-name-segment {
+                        margin-left: 0;
+                        margin-right: 0.2rem;
+                        line-height: normal;
+                    }
+                    .block-name-param {
+                        border-radius: 1rem;
+                        padding: 0.2rem 0.4rem;
+                        background-color: rgba(0, 0, 0, 0.3);
```

Comment:
Should we make or use one of the color variables here? I don't have a strong preference, but am curious if there might be a use case elsewhere for a slightly translucent black.